src/CodeCheckers/ApostropheChecker.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace CodeCheckers;
+
+use Nette\Object;
+use Nette\Utils\Strings;
+
+
+
+class ApostropheChecker extends Object
+{
+
+	public static function createChecker()
+	{
+		return function ($checker, $s) {
+			if (!$checker->is('php') && !$checker->is('phpt')) {
+				return;
+			}
+
+			$s = implode("\n", array_map(function ($line) use ($checker) {
+				$line = rtrim($line);
+				if (Strings::contains($line, "'")
```

Comment:
teďka jsem ti to mergoval a ještě jsem to prochazel u sebe, protože jsem fixoval nejaky konflikty po rebasu... a zjistil jsem ze tenhle tvuj detektor vubec nepocita napriklad s dvojteckama, nebo s `{`, takže nefixne `"PARTIAL e.{id}"` atd...